src/types/auth.ts

```diff
@@ -1,18 +1,14 @@
-export interface AuthInterface {
-  secret: string;
-  device_id: string;
-}
+export type ApiToken = string;
 
 export type AuthToken = {
   id: number;
-  device_id: string;
   user_id: number;
   user_agent: string;
   application?: string;
 };
 
-export type AuthTokenWithSecret = AuthToken & {
-  secret: string;
+export type AuthTokenWithToken = AuthToken & {
+  token: ApiToken;
 };
```

Comment:
This type doesn't fully match the object we would return during our transition period, but it does match the object that would be returned after we remove the fallbacks/transition